assets/app/view/game_page.rb

```diff
@@ -410,6 +410,10 @@ def render_action
           h(Game::Round::Merger, game: @game)
         elsif current_entity_actions.include?('buy_shares') && @game.current_entity&.player?
           h(Game::Round::Stock, game: @game)
+        elsif current_entity_actions.include?('bid')
+          h(Game::Round::Auction, game: @game, user: @user)
+        elsif current_entity_actions.include?('choose')
+          h(Game::Round::Choices, game: @game)
```

Comment:
Won't this break existing games that have a "choose" action in an OR? There are several that do. Not sure about "bid"

Reply:
If you want a "Choices" round, why are you doing it from an Operating round?

Reply:
When NdeM privatization occurs, the NdeM has one more operating round that runs normally, and then there is a phase where the NdeM tokens are auctioned off.  I am using "Choices" to allow the current player to pick which NdeM token they want to start an auction for.  I may have not been understanding the code, but it looked like it was not limited to using "Game::Round::foo" within "Engine::Round::foo".  

If there are other games that have something that chooses a token from the map from within an OR, I can conform to what they did.

Reply:
1856 does this during CGR formation (mid-OR) - if the CGR forms with too many tokens the president must choose which ones to remove

https://github.com/tobymao/18xx/blob/master/lib/engine/game/g_1856/step/remove_tokens.rb

Reply:
Thank you - I will take a look.

Reply:
My point is that you are forcing a different UI::Round when "choose" is an action. That will break games that rely on "choose" still using UI::Round::Operating

Reply:
As it turns out - I had already changed it to remove_token - I had just forgotten.  Removing "choose".  I do not think that what I have would work if I removed the "bid" one.  What would be the best way to make sure I didn't break anything else with that change?

Reply:
Look at UI::Round::Operating and see if it checks for a "bid" action. If it does, you are very likely breaking other games.

Reply:
Do you mean Game::Round::Operating?  If so, it doesn't check for a "bid" action.

Reply:
View::Game::Round::Operating (assets/app/view/game/round/operating.rb)

Reply:
It's a little strange to use View::Game::Round::Auction for an operating round.

Reply:
Would the alternative be to implement "bid" inside View::Game::Round::Operating?  I did it this way because I thought there would be a lot of duplication between the code I'd be adding to Operating and Auction.

Reply:
I was also basing it on the code using View::Game::Round::Merger and View::Game::Round::Stock inside an operating round that was already there.

Reply:
It's probably OK, I'm just being paranoid.

Reply:
Paranoia is good.  Is there anything else I should address for this PR?